README-local-development.md

```diff
@@ -38,6 +34,70 @@ Jekyll will build the site and start a web server, which can be viewed in your b
 
 For more info on how to use Jekyll, check out [their docs](https://jekyllrb.com/docs/usage/).
 
-## Editing and Contributing
+# Editing Docs Locally
+
+The docs site includes Bootstrap 3 JS and CSS, so you'll have access to all of its [reusable components](https://v4-alpha.getbootstrap.com/components/alerts/).
+
+All docs live in folders named after the version of CircleCI. The only two you need to worry about are `jekyll/_cci1` and `jekyll/_cci2`. These correspond to CircleCI Classic and CircleCI 2.0, respectively.
+
+1. Create a branch and switch to it:
+
+    `git checkout -b <branch-name`
+
+2. Add or modify Markdown files in these directories according to our [style guide](CONTRIBUTING#style-guide).
+
+3. When you're happy with your changes, commit them with a message summarizing what you did:
+
+    `git commit -am "commit message"`
+
+4. Push your branch up:
+
+    `git push origin <branch-name>`
+
+## Adding New Articles
+
+New articles can be added to the [jekyll/_cci2](https://github.com/circleci/circleci-docs/tree/master/jekyll/_cci2) directory in this repo.
+
+When you make a new article, you'll need to add [**front matter**](https://jekyllrb.com/docs/frontmatter/). This contains metadata about the article you're writing and is required so everything works on our site.
+
+Front matter for our docs will look something like:
+
+```
+---
+layout: classic-docs
+title: "Your Doc Title"
+short-title: "Short Title"
+categories: [category-slug]
+order: 10
+---
+```
+
+`layout` describes visual settings shared across our docs. `title` will appear at the top of your article and appear in hyphenated form for the URL. `short-title` will display in the sidebar on the left underneath the article's category.
+
+The list of available categories can be found in [categories.yml](https://github.com/circleci/circleci-docs/blob/master/jekyll/_data/categories.yml).
```

Comment:
hmm, we aren't using this categories.yml convention in 2.0 docs anymore 
anymore. So, we should say that and explain that navigation links are manually added to 
the landing pages and that this is a decision the writer must make. (Or that the docs leads can make for the contributor).

Reply:
Does that mean we're also not using the `order` convention?

Reply:
Good catch, though! :)